### ISSUE TITLE
tutorial needs more explanations: PacBio

### ISSUE THREAD

**Issue by andzandz11** [DormanLab/AmpliCI#7]
Can you please expand the tutorial and explain how to convert PacBio quality encoding to what is required by your tool? seqkit does not accept PacBio scores
Thanks!

**Comment by kdorman** [comment]
Hello,

What are PacBio quality scores? Are they Phred-encoded error probabilities? What range? We have some in-house scripts that might be able to handle PacBio quality scores are in your dataset, but this may not be your last difficult with using AmpliCI on PacBio reads. I'll point out a few facts about AmpliCI that may prevent it from working well for PacBio reads. 

1. AmpliCI assumes indels are rare. It doesn't even consider them at first (for speed) and when it does consider them, it penalizes them quite strictly. You can change the gap penalty (--scores). Theoretically, you can also try gapped alignments (--align), but it would be slow and is not much tested.

2. It assumes error-free molecules are seen at least twice, so for longer molecules with higher error rates, it may fail to get started without this necessary observed replication.

3. Related to 1, since it assumes few indels, it expects equal-length reads and will throw warnings, but still run, with reads that are not the same length. Its behavior for reads of highly varying length has not been studied.

In short, we have never tested it on PacBio reads, but perhaps it can work for high quality CCS reads. We are willing to explore if you can point us to a sample dataset or point to one like yours that is publicly available.

Thanks for your interest!

Best,
Karin
